packages/env/lib/build-docker-compose-config.js

```diff
@@ -167,6 +167,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 	// Set the default ports based on the config values.
 	const developmentPorts = `\${WP_ENV_PORT:-${ config.env.development.port }}:80`;
 	const testsPorts = `\${WP_ENV_TESTS_PORT:-${ config.env.tests.port }}:80`;
+	const testsMysqlPorts = `\${WP_ENV_TESTS_MYSQL_PORT:-${ config.env.tests.mysqlPort ?? '' }}:3306`;
```

Comment:
If we're going to support customizing the port like this let's go ahead and also support the development environment's MySQL port too.